Libs/Widgets/Testing/Cpp/ctkAxesWidgetTest1.cpp

```diff
@@ -84,6 +85,19 @@ int ctkAxesWidgetTest1(int argc, char * argv [] )
             << spy[0].at(0).toInt() << " " << spy[1].at(0).toInt() << std::endl;
     return EXIT_FAILURE;
     }
+
+  QStringList axesLabels;
+  axesLabels <<  "W";
+  axesLabels <<  "E";
+  axesLabels <<  "S";
+  axesLabels <<  "N";
+  axesLabels <<  "Z";
+  axesLabels <<  "z";
+  if (!axes.setAxesLabels(AxesLabels))
+    {
+    return EXIT_FAILURE; 
+    }
```

Comment:
Check also that axes.axesLabels() return the expected list.

You could include the new `ctkCoreTestingMacros.h` and use `CHECK_QSTRINGLIST()`